TKGame/Components/Concrete/Weapon_GraphicsComponent.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Microsoft.Xna.Framework.Graphics;
+using Microsoft.Xna.Framework.Input;
+using Myra.Graphics2D.UI;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TKGame.BackEnd;
+using TKGame.Components.Interface;
+using TKGame.Content.Weapons;
+
+
+namespace TKGame.Components.Concrete
+{
+    internal class Weapon_GraphicsComponent : GraphicsComponent
+    {
+        public Texture2D weaponTexture = Art.PlayerTexture;
+        private VerticalSplitPane VSP { get; set; }
+        void GraphicsComponent.Update(Entity entity)
+        {
+
+            //Adds Weapon To Player Sprite
+            if (Input.KeyboardState.IsKeyDown(Keys.D1))
+            {
+                entity.weapon.isActiveSword = true;
+                entity.weapon.isActiveSpear = false;
+                entity.weapon.isActiveAxe = false;
+            }
+            //else if (Input.KeyboardState.IsKeyDown(Keys.D2))      // Will Crash if you use the number keys, some problem with moving spritebatch draw into player graphics component;
```

Comment:
Problems when i pulled dev into my repo, crashes when you turn on caps lock and then hit 2 or 3 and says that entity texture is null.